sensor/kubernetes/fake/flows.go

```diff
@@ -174,7 +174,17 @@ func getNetworkProcessUniqueKeyFromProcess(process *storage.ProcessSignal) *stor
 }
 
 func getRandomOriginator(containerID string) *storage.NetworkProcessUniqueKey {
-	process := processPool.getRandomProcess(containerID)
+	var process *storage.ProcessSignal
+	var percentMatchedProcess float32 = 0.5
+	p := rand.Float32()
+	if p < percentMatchedProcess {
+		// There is a chance that the process has been filtered out or hasn't gotten to
+		// the central-db for some other reason so this is not a guarantee that the
+		// process is in the central-db
+		process = processPool.getRandomProcess(containerID)
+	} else {
+		process = getGoodProcess(containerID)
+	}
```

Comment:
In half of the cases the process for the listening endpoint will not exist in the central database. This will help test these cases, which is a common occurrence.